File: Problem 327P1/0327_Problem_1.cs
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 112 ms

public int MaximumCount(int[] nums) {
    return  Math.Max(
        nums.Count(x => x < 0),
        nums.Count(x => x > 0));
}

File: Problem 327P1/0327_Problem_1.js
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 69 ms

var maximumCount = function(nums) {
    return Math.max(
        nums.filter(e => e > 0).length,
        nums.filter(e => e < 0).length)
};
